README.md

```diff
@@ -12,6 +12,8 @@
 
 - **path_prefix** prefix of target keys (string, required)
 
+- **direct_path_prefix_object** flag to enable treating `path_prefix` as a single object key (boolean, default to `false`). Since list-objects operation on S3 is eventually consistent, using this for a  more reliable way to retrieve a single object when you know the exact object key.
+
```

Comment:
`path_prefix` + `direct_path_prefix_object` is confusing for specifying a single object. How about adding `path` to specify an object, and let `path_prefix` scan multiple files?

Reply:
For comment, to be precise, it's not something related to 'eventual consistency'. Consistency here is "When you can successfully run GetObject, ListObject returns the object" + "After a PutObject, ListObject returns the object".
AFAIK S3 does not say it's consistent in specification level, but from observation it's consistent in implementation level. RiakCS is not consistent. Other S3 compatible APIs also can be inconsistent I guess.

This direct `path` option should be for skipping ListObject call to avoid this consistency issue.

Reply:
Yes, S3 is indeed eventually consistent implementation-wise https://forums.aws.amazon.com/thread.jspa?messageID=687028&#687028, I couldn't find anywhere RiakCS state its object listing consistency. @trung-huynh could you help me to remove this false assumption on your new PR (`direct_path_prefix_object` would be removed anyway)?. @nahi Could you help me to add @trung-huynh as a collaborator to this project so he could work on the same remote instead of a fork?

Reply:
@huylenq I added @trung-huynh as a member of this repository.

Reply:
@huylenq Oh, I didn't know AWS confirmed that S3 also can not satisfy  "When you can successfully run GetObject, ListObject returns the object". Good to know, thanks.